modules/edu.gemini.seqexec.server/src/main/scala/edu/gemini/seqexec/server/Executor.scala

```diff
@@ -14,13 +14,14 @@ import Scalaz._
 import scalaz.concurrent.Task
 
 object Step {
+  val dhsClient = DhsClientSim(LocalDate.now())
+
   type Step = EitherT[Task, NonEmptyList[SeqexecFailure], StepResult]
 
-  def parConfig(config: List[SeqAction[ConfigResult]]):
-    EitherT[Task, NonEmptyList[SeqexecFailure], List[ConfigResult]] =
-      EitherT(Nondeterminism[Task].gather(config.map(_.run)).map(_.map(_.validationNel).sequenceU.disjunction))
+  private def parConfig(config: List[SeqAction[ConfigResult]]): EitherT[Task, NonEmptyList[SeqexecFailure], List[ConfigResult]] =
+    EitherT(Nondeterminism[Task].gather(config.map(_.run)).map(_.map(_.validationNel).sequenceU.disjunction))
```

Comment:
Yes, `_.map(_.validationNel).sequenceU` can be written `_.traverseU(_.validationNel)`.